src/components/CheckIn/CheckInItems.js

```diff
@@ -98,6 +99,14 @@ class CheckInItems extends Component {
     addPiece(values);
   }
 
+  addPieceModalCheckIn = () => {
+    const { checkInItem, stripes: { store } } = this.props;
+    const values = getFormValues('AddPieceModalForm')(store.getState());
```

Comment:
I still think it should be in `AddPieceModal` class as a class property

Reply:
yeah, you should pass `addPieceModalCheckIn` to your component and pass `values` there